### PR TITLE
feat(SearchForm): Radio button filters

### PR DESCRIPTION
**Pull request by KaylaBrady**
ticket: https://app.asana.com/0/1200180014510248/1205198429752070/f

### CODE REVIEW

File: assets/src/components/searchForm.tsx [hunk 1]
```diff
@@ -299,7 +271,11 @@ export const SearchForm = ({
               id={autocompleteId}
               controlName="Search Suggestions"
               maxElementsPerGroup={5}
-              searchFilters={filters}
+              searchFilters={
+                property === "all"
+                  ? allFiltersOn
+                  : { ...allFiltersOff, [property]: true }
+              }
```
Comment (firestack) [inline assets/src/components/searchForm.tsx:278]: praise: this is a neat way of doing this!

File: assets/tests/components/searchForm.test.tsx [hunk 2]
```diff
@@ -252,21 +258,48 @@ describe("SearchForm", () => {
...
+          <StateDispatchProvider
+            state={{
+              ...initialState,
+              searchPageState: {
+                ...initialState.searchPageState,
+                query: emptySearchQueryFactory.build({
+                  text: "123",
+                  property: selectedProperty,
+                }),
+              },
```
Comment (firestack) [inline assets/tests/components/searchForm.test.tsx:288]: nit(non-blocking): would it be possible to use the `stateFactory` here instead of `initialState`?

**Review by firestack — APPROVED** [review]
Nothing stuck out to me, I think this looks good!

**Review by lemald — APPROVED** [review]
👍

**Pull request merged**